cmd/fluxctl/release_cmd.go

```diff
@@ -54,20 +54,20 @@ func (opts *workloadReleaseOpts) Command() *cobra.Command {
 	AddOutputFlags(cmd, &opts.outputOpts)
 	AddCauseFlags(cmd, &opts.cause)
 	cmd.Flags().StringVarP(&opts.namespace, "namespace", "n", "default", "Workload namespace")
-	// Note: we cannot define a shorthand for --workload since it clashes with the shorthand of --watch
-	cmd.Flags().StringSliceVarP(&opts.workloads, "workload", "", []string{}, "List of workloads to release <namespace>:<kind>/<name>")
+	cmd.Flags().StringSliceVarP(&opts.workloads, "workload", "w", []string{}, "List of workloads to release <namespace>:<kind>/<name>")
 	cmd.Flags().BoolVar(&opts.allWorkloads, "all", false, "Release all controllers")
 	cmd.Flags().StringVarP(&opts.image, "update-image", "i", "", "Update a specific image")
 	cmd.Flags().BoolVar(&opts.allImages, "update-all-images", false, "Update all images to latest versions")
 	cmd.Flags().StringSliceVar(&opts.exclude, "exclude", []string{}, "List of controllers to exclude")
 	cmd.Flags().BoolVar(&opts.dryRun, "dry-run", false, "Do not release anything; just report back what would have been done")
 	cmd.Flags().BoolVar(&opts.interactive, "interactive", false, "Select interactively which containers to update")
 	cmd.Flags().BoolVarP(&opts.force, "force", "f", false, "Disregard locks and container image filters (has no effect when used with --all or --update-all-images)")
-	cmd.Flags().BoolVarP(&opts.watch, "watch", "w", false, "Watch rollout progress during release")
+	// Note: we cannot define a shorthand for --watch since it clashes with the shorthand of --workload
+	cmd.Flags().BoolVarP(&opts.watch, "watch", "", false, "Watch rollout progress during release")
```

Comment:
I agree with this change, but the problem is that it breaks backwards compatibility.

@squaremo @hiddeco thoughts?

Reply:
I share your opinion, and am quite scared people are using it in their CI. 

Following strict semver it would mean a major release :weary:

Reply:
Yeah, I am afraid this has to wait for some more major overhaul, so it can be made backward-compatibly, _fuir vers l'avant_.

Reply:
This means we need to make an update on #1985 that just got merged though, as this mentions `-w` as being `--workload` and not `--watch`.

Reply:
As an example of such an overhaul: new subcommands which have a consistent set of arguments, e.g.,

    fluxctl update image --all-workloads --watch foobar:v1